buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -1,28 +1,28 @@
 object Versions {
-    const val KOTLIN = "1.7.20"
+    const val KOTLIN = "1.8.10"
     const val ATOMIC_FU = "0.18.5"
     const val ANDROID_GRADLE_PLUGIN = "7.3.1"
-    const val JETPACK_COMPOSE_COMPILER = "1.3.2"
-    const val JETPACK_COMPOSE_RUNTIME = "1.3.2"
+    const val JETPACK_COMPOSE_COMPILER = "1.4.3"
+    const val JETPACK_COMPOSE_RUNTIME = "1.3.3"
     const val JETPACK_COMPOSE_FOUNDATION = "1.3.1"
-    const val JETPACK_COMPOSE_UI_TOOLING = "1.3.2"
+    const val JETPACK_COMPOSE_UI_TOOLING = "1.3.3"
     const val JETPACK_COMPOSE_MATERIAL = "1.3.1"
-    const val JETPACK_COMPOSE_MATERIAL_3 = "1.1.0-alpha03"
+    const val JETPACK_COMPOSE_MATERIAL_3 = "1.1.0-alpha07"
     const val COIL = "2.0.0-rc03"
-    const val KTLINT = "10.3.0"
+    const val KTLINT = "11.2.0"
     const val KOTLINX_SERIALIZATION = "1.4.1"
```

Comment:
[Kotlinx.serialization 1.5.0](https://github.com/Kotlin/kotlinx.serialization/releases/tag/v1.5.0) is also out with kotlin 1.8.10 support 🙂